core/src/main/scala/org/apache/spark/deploy/history/HistoryPage.scala

```diff
@@ -35,8 +35,9 @@ private[history] class HistoryPage(parent: HistoryServer) extends WebUIPage("")
     val lastUpdatedTime = parent.getLastUpdatedTime()
     val providerConfig = parent.getProviderConfig()
     val content =
-      <script src={UIUtils.prependBaseUri(request, "/static/historypage-common.js")}></script> ++
-      <script src={UIUtils.prependBaseUri(request, "/static/utils.js")}></script>
+      <script type="module"
+              src={UIUtils.prependBaseUri(request, "/static/historypage-common.js")} /> ++
```

Comment:
Unfortunately, this seems to break `Spark History Server` currently.

Reply:
Let me fix it. Thank you @dongjoon-hyun

Reply:
I made a draft, @yaooqinn .

Reply:
https://github.com/apache/spark/pull/44930